crypto/vrf/vrf.go

```diff
@@ -38,178 +34,109 @@ const (
 	ProofSize        = 32 + 32 + intermediateSize
 )
 
-var (
-	ErrGetPubKey = errors.New("[vrf] Couldn't get corresponding public-key from private-key")
-)
+// PrivateKey represents a Curve25519 private key.
+type PrivateKey [PrivateKeySize]byte
```

Comment:
On the long run we should even switch to using an interface which completely hides the underlying representation (similar to this: https://github.com/google/keytransparency/blob/master/core/vrf/vrf.go)